tests/ui/src/utilities/s3Helper.ts

```diff
@@ -1,28 +1,60 @@
+import { S3Client, PutObjectCommand, GetObjectCommand, DeleteObjectCommand } from '@aws-sdk/client-s3';
+import { readFile } from 'fs/promises';
+import { createWriteStream } from 'fs';
+import { pipeline, Readable } from 'stream';
+import { promisify } from 'util';
 
 const bucket = `nhse-uec-cm-ui-test-bucket-${process.env.ENV}${process.env.WORKSPACE}`;
 
-export function addObject(sourceFilePath: string, targetFile: string) {
-  return require('child_process').execSync(
-    `aws s3api put-object --bucket ${bucket} --key ${targetFile} --body ${sourceFilePath}`
-)};
+const client = new S3Client({ region: process.env.REGION });
 
-export function getObject(sourceFile: string, targetFile?: string) {
+export async function addObject(sourceFilePath: string, targetFile: string) {
+  try {
+    const fileContent = await readFile(sourceFilePath);
+    const command = new PutObjectCommand({
+      Bucket: bucket,
+      Key: targetFile,
+      Body: fileContent,
+    });
+
+    await client.send(command);
+  } catch (error) {
+    console.error('Error', error);
+  }
+};
```

Comment:
The original `addObject` function would crash out if it got a non-zero exit code, so I'd just remove the wrapping try-catch block and let this error bubble up and fail the test